melos.yaml

```diff
@@ -2,5 +2,8 @@ name: stripe
 packages:
   - /**
 scripts:
+  analyze: melos exec -- flutter analyze
   format: melos exec -- flutter format .
+  get: melos exec -- flutter pub get
```

Comment:
Is this required since this is what `melos bootstrap` is for and it will correctly link local packages together?